jwst/regtest/test_dark.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from astropy.io.fits.diff import FITSDiff
+
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+from jwst.stpipe import Step
+
+@pytest.mark.bigdata
+def test_miri_dark_cal( _jail, rtdata, fitsdiff_default_kwargs ):
+    rtdata.get_data("miri/image/jw00001001001_01101_00001_MIRIMAGE_uncal.fits")
+
+    collect_pipeline_cfgs("config")
+    args = ["config/calwebb_dark.cfg", rtdata.input,
+            "--steps.rscd.save_results=True"]
```

Comment:
I see the results of the ``rscd`` step being saved here, but I don't see them being used anywhere, such as in the comparison of outputs. It looks like the only comparison being done is on the main output of the pipeline, which is the _dark product.